src/Gui/NavigationStyle.cpp

```diff
@@ -353,171 +323,71 @@ SbBool NavigationStyle::lookAtPoint(const SbVec2s screenpos)
 
     SbVec3f hitpoint;
     hitpoint = picked->getPoint();
-    lookAtPoint(hitpoint);
+    this->rotationCenterFound = false;
```

Comment:
The public function `void lookAtPoint(const SbVec3f&)` shouldn't be simply removed. You can implement it as
```
void NavigationStyle::lookAtPoint(const SbVec3f& pos)
{
    this->rotationCenterFound = false;
    translateCamera(hitpoint - getFocalPoint());
}
```
and leave its call in `lookAtPoint(const SbVec2s screenpos)`

Reply:
Resolved in 26ea9614e64b923008f29cbb06d4b6f45efc0b37